docs/apache-airflow/dag-run.rst

```diff
@@ -264,9 +264,33 @@ Example of a parameterized DAG:
         bash_command="echo value: {{ dag_run.conf['conf1'] }}",
         dag=dag,
     )
+    
+**Note**: The parameters from ``dag_run.conf`` can only be used in a template field of an operator.
```

Comment:
Let’s make this a normal paragraph; I feel it reads better combined with the newly added content. Something like this:

> Note that the parameters from […]. Example of a parametrized DAG […]: